src/NServiceBus.AcceptanceTests/Performance/TimeToBeReceived/When_TimeToBeReceived_used_with_unobtrusive_mode.cs

```diff
@@ -0,0 +1,119 @@
+﻿namespace NServiceBus.AcceptanceTests.Performance.TimeToBeReceived
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+
+    public class When_TimeToBeReceived_used_with_unobtrusive_mode : NServiceBusAcceptanceTest
```

Comment:
using ttbr with transports that don't support it will throw an exception on the downstreams. Or is this test already excluded through it's folder?